src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -816,12 +816,14 @@ private void ImportProcedure(string library, ProcedureNode proc)
                             defaultArgumentNode = binaryExpr.RightNode;
                         }
                     }
+                    string shortName = null;
                     if (defaultArgumentNode != null)
                     {
+                        shortName = defaultArgumentNode.ToString();
```

Comment:
The only other option would be is to rewrite the resultant AST again to shorten the names. In other words have another class similar to `ElementRewriter` for shortening `IdentifierListNode` ASTs, which I personally think is complicating things a lot. If this is what is needed then I'll go ahead with it.